### PR TITLE
global: setup circulation UI structure

### PR DESCRIPTION
**Pull request by mvesper**
- Adds the first draft of the circulation UI, handling the basic admin
  functionality.

### CODE REVIEW

File: invenio_circulation/ext.py [hunk 1]
```diff
@@ -40,6 +41,7 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
+        app.register_blueprint(blueprint)
```
Comment (jirikuncar) [inline invenio_circulation/ext.py]: please use entry point group `invenio_base.blueprints` instead

Comment (jirikuncar) [inline invenio_circulation/ext.py]: and move it to after https://github.com/inveniosoftware/invenio-circulation/pull/29/files#diff-86d2f29b7151f1dfb6b158f5f495ba35R95

File: invenio_circulation/templates/invenio_circulation/index.html [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
+
+{%- block javascript %}
+  {{ super() }}
+  {% assets "invenio_search_ui_search_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+{%- endblock javascript %}
+
 {%- block page_body %}
-TODO: Example template, please remove if you do not need it.
-{{_('Welcome to %(module_name)s', module_name=module_name)}}
+WUHU :)
```
Comment (PXke) [inline invenio_circulation/templates/invenio_circulation/index.html]: woot ?

Comment (mvesper) [inline invenio_circulation/templates/invenio_circulation/index.html]: WIP :)

**Comment by jirikuncar** [comment]
@mvesper can you add `InvenioCirculation` to both `invenio_base.apps` and `invenio_base.api_apps`?

File: invenio_circulation/bundles.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    output='gen/circulation.%(version)s.css',
+)
+
+js = NpmBundle(
+    'node_modules/angular/angular.js',
+    'js/circulation/app.js',
+    filters='jsmin',
+    output='gen/circulation.%(version)s.js',
+    npm={
+        "angular": "~1.4.8",
```
Comment (jirikuncar) [inline invenio_circulation/bundles.py]: `s/"/'/`

**Pull request merged**